lib/std/time.zig

```diff
@@ -33,16 +33,50 @@ pub fn timestamp() u64 {
 /// Get the posix timestamp, UTC, in milliseconds
 /// TODO audit this function. is it possible to return an error?
 pub fn milliTimestamp() u64 {
+    return @divFloor(nanoTimestamp(), millisecond);
+}
+
+const DarwinTimeStart = struct {
+    timebase: os.darwin.mach_timebase_info_data,
+    inittime: os.darwin.timespec,
+    initclock: u64,
+};
+
+var global_timestart: DarwinTimeStart = undefined;
+var init_global_timestart_once = std.once(init_global_timestart);
+
+pub fn init_global_timestart() void {
```

Comment:
I think this shouldn't be `pub`? If we want to permit people to pre-load it then make `init_global_timestart_once` `pub`.